css/src/tokens/themes.scss

```diff
@@ -6,6 +6,8 @@ $themes: (
 		'text': $palette-grey-120,
 		'text-subtle': $palette-grey-90,
 		'text-invert': $palette-white,
+		'text-purple': $palette-pink,
```

Comment:
Do we need these? Should they be removed? If not, can we use a theme concept instead of a specific color?

Reply:
Removed